src/hypothesis/extra/datetime.py

```diff
@@ -15,111 +15,120 @@
 #
 # END HEADER
 
+"""This module provides time and date related strategies.
+
+It depends on the ``pytz`` package, which is stable enough that almost any
```

Comment:
I feel like it would be better to put the timezones strategy into a `hypothesis.extra.pytz` package and let htis one become entirely deprecated.